packages/ooth-jwt/src/index.ts

```diff
@@ -59,22 +42,50 @@ export default function({
   sharedSecret,
   privateKey,
   publicKey,
-  algorithm = 'RS256'
+  algorithm = 'RS256',
+  includeProfile = false,
 }: Options): void {
-  if(sharedSecret === undefined && privateKey === undefined) {
+  async function getToken(userId: string, iat: number, tokenExpiry: number, options: TokenOptions): Promise<string> {
+    const data: {[key: string]: any} = {
+      iat,
+      exp: iat + tokenExpiry,
+      _id: userId,
+    };
+
+    if (options.includeProfile) {
+      data.profile = ooth.getProfile(await ooth.getUserById(userId));
```

Comment:
Use data.user instead (it would be in line with the convention used by `ooth-user`)